docs/minimal-test.md

```diff
@@ -0,0 +1,318 @@
+---
+layout: minimal
+title: Minimal layout test
```

Comment:
The body of this file appears to be a copy of `docs/index-test.md`.
1. Why is all this Markdown code relevant for testing any layout? It appears to be testing only the styling, which is shared between layouts, it seems.
2. If Markdown code needs to be shared between two Jekyll files, it can be moved to an include (with extension `.md`).

Reply:
Great point; hadn't given it much thought. Let me replace it with something that is more suitable (a smaller text that explains the purpose of the file).